### ISSUE TITLE
Giving permissions when containing page is shown breaks the app

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-plugin-barcode-scanner#8]
### Problem description

When a new page is created containing the scanner view and its `start()` method gets called, a permission dialog will be shown when the camera permission has not been given yet. When accepting the permission request, the app content will slide to the right and the user will be left with an unusable white screen.

### Environment

- Tabris.js version: 2.4.0
- Device: Samsung Galaxy S8
- OS: Android 8

### Code snippet

```javascript
const {Button, Page, NavigationView, ui} = require('tabris');

let navigationView;
let scannerView;
let button;

ui.contentView.append(
  navigationView = new NavigationView({left: 0, top: 0, right: 0, bottom: 0}),
  button         = new Button({centerX: 0, centerY: 0, text: 'Open scanner page'})
                    .on({tap: () => openScannerPage()})
)

function openScannerPage() {
  button.dispose();
  navigationView.append(
    new Page().append(
      scannerView = new esbarcodescanner.BarcodeScannerView({left: 0, right: 0, top: 0, bottom: 0})
    )
  )
  .appendTo(ui.contentView);
  scannerView.start();
}

```

**Comment by mpost** [comment]
We could reproduce the issue and are looking into a solution.

**Comment by mpost** [comment]
The issue is caused by a regression in tabris-android 2.4.0. It is fixed in 2.4.1. Closing this issue.
